action-server/actions/answers.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from typing import List
+
+import requests
+
+ANSWERS_PATH = "answers"
+QUESTION_KEY = "question"
+
+
+class QaResponse(BaseModel):
+    answers: List[str]
+    status_code: int
+
+
+class QaProtocol:
+    def __init__(self, base_url: str):
+        self.base_url = base_url
+
+    def get_response(self, question: str) -> QaResponse:
+        response = requests.get(
+            f"{self.base_url}/{ANSWERS_PATH}", params={QUESTION_KEY: question}
+        )
+        return QaResponse(**response.json(), status_code=response.status_code)
```

Comment:
Since I'm writing the form which will call it, we concluded this part could only expose a function that I can call inside (as long as Vincent tested it somehow before merging, of course)

Reply:
I've technically already tested this in an action and a form, but I removed it as that will go in Karine's PR. That's why I have the __main__.py to test these requests for now.

Reply:
This was tested via a custom action and form and the \_\_main\_\_.py is meant to test the queries for now.